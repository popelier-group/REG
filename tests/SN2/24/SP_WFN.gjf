# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.34607400   -0.00006400    0.00011700
 H                 -1.02602800   -0.87641100   -0.54594700
 H                 -1.02596000    0.91100400   -0.48576500
 H                 -1.02608900   -0.03481900    1.03210500
 Br                 1.66627900    0.00001100   -0.00001400
 Cl                -3.20985500    0.00000400   -0.00002100

frame.wfn
