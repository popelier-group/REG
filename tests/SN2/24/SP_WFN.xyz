
6

C       -1.217789    0.013774    0.129429
H       -0.813116   -0.536865    0.967066
H       -1.001166   -0.474568   -0.810356
H       -0.884367    1.042086    0.130190
Br        1.777514   -0.019957   -0.188841
Cl       -3.071035    0.034424    0.326233
