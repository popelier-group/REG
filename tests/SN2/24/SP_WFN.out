 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.24/Gau-66773.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.24/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     66774.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.34607  -0.00006   0.00012 
 H                    -1.02603  -0.87641  -0.54595 
 H                    -1.02596   0.911    -0.48577 
 H                    -1.02609  -0.03482   1.03211 
 Br                    1.66628   0.00001  -0.00001 
 Cl                   -3.20986   0.       -0.00002 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.346074   -0.000064    0.000117
      2          1           0       -1.026028   -0.876411   -0.545947
      3          1           0       -1.025960    0.911004   -0.485765
      4          1           0       -1.026089   -0.034819    1.032105
      5         35           0        1.666279    0.000011   -0.000014
      6         17           0       -3.209855    0.000004   -0.000021
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.081018   0.000000
     3  H    1.081018   1.788428   0.000000
     4  H    1.081017   1.788442   1.788438   0.000000
     5  Br   3.012353   2.883518   2.883403   2.883631   0.000000
     6  Cl   1.863781   2.415624   2.415629   2.415643   4.876134
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.217789    0.013774    0.129429
      2          1           0       -0.813116   -0.536865    0.967066
      3          1           0       -1.001166   -0.474568   -0.810356
      4          1           0       -0.884367    1.042086    0.130190
      5         35           0        1.777514   -0.019957   -0.188841
      6         17           0       -3.071035    0.034424    0.326233
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         156.7780324           0.8340368           0.8340368
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       170.5633794158 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.60D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40083757     A.U. after   11 cycles
            NFock= 11  Conv=0.69D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55616-101.40493 -62.20927 -56.02706 -56.02699
 Alpha  occ. eigenvalues --  -56.02699 -10.07923  -9.31927  -8.41807  -7.08287
 Alpha  occ. eigenvalues --   -7.07455  -7.07455  -6.24117  -6.24077  -6.24077
 Alpha  occ. eigenvalues --   -2.35356  -2.35342  -2.35342  -2.35311  -2.35310
 Alpha  occ. eigenvalues --   -0.70443  -0.54094  -0.48926  -0.30428  -0.30427
 Alpha  occ. eigenvalues --   -0.26736  -0.17195  -0.17195  -0.05454  -0.05408
 Alpha  occ. eigenvalues --   -0.05408
 Alpha virt. eigenvalues --    0.11098   0.12607   0.14972   0.14972   0.17498
 Alpha virt. eigenvalues --    0.18543   0.19603   0.19603   0.20799   0.20800
 Alpha virt. eigenvalues --    0.21772   0.22596   0.22597   0.24857   0.28616
 Alpha virt. eigenvalues --    0.30991   0.30992   0.37018   0.40614   0.53783
 Alpha virt. eigenvalues --    0.53784   0.55524   0.58750   0.58750   0.59430
 Alpha virt. eigenvalues --    0.62087   0.62088   0.66186   0.76883   0.76883
 Alpha virt. eigenvalues --    0.79563   0.84284   0.87789   0.87789   0.89126
 Alpha virt. eigenvalues --    0.89127   0.95779   0.97364   0.97364   1.01515
 Alpha virt. eigenvalues --    1.01515   1.04007   1.11900   1.26806   1.26807
 Alpha virt. eigenvalues --    1.27047   1.57105   1.67929   1.67929   1.85625
 Alpha virt. eigenvalues --    1.85625   1.88226   2.08367   2.21174   2.21175
 Alpha virt. eigenvalues --    2.28009   2.28010   2.31335   2.45898   2.45898
 Alpha virt. eigenvalues --    2.58054   2.60214   2.64328   2.64328   2.67780
 Alpha virt. eigenvalues --    2.67780   3.13747   3.28997   3.87476   3.87478
 Alpha virt. eigenvalues --    4.39542   4.39544   4.40428   4.40429   4.54121
 Alpha virt. eigenvalues --    6.76989   7.80338   7.80343   7.86790   9.84402
 Alpha virt. eigenvalues --   24.03930  25.99166  25.99166  26.49693  48.08286
 Alpha virt. eigenvalues --  215.82986 290.01204 290.01205 290.071031020.84257
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.704519   0.407505   0.407501   0.407511  -0.067532  -0.203594
     2  H    0.407505   0.521704  -0.038647  -0.038650  -0.002123  -0.042941
     3  H    0.407501  -0.038647   0.521691  -0.038649  -0.002127  -0.042931
     4  H    0.407511  -0.038650  -0.038649   0.521712  -0.002116  -0.042952
     5  Br  -0.067532  -0.002123  -0.002127  -0.002116  35.925010   0.061619
     6  Cl  -0.203594  -0.042941  -0.042931  -0.042952   0.061619  17.281617
 Mulliken charges:
               1
     1  C   -0.655909
     2  H    0.193152
     3  H    0.193161
     4  H    0.193144
     5  Br  -0.912730
     6  Cl  -0.010818
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.076452
     5  Br  -0.912730
     6  Cl  -0.010818
 Electronic spatial extent (au):  <R**2>=           1152.7795
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -4.0561    Y=              0.0457    Z=              0.4310  Tot=              4.0792
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -74.9194   YY=            -44.3854   ZZ=            -44.7262
   XY=              0.3425   XZ=              3.2441   YZ=             -0.0364
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -20.2424   YY=             10.2916   ZZ=              9.9508
   XY=              0.3425   XZ=              3.2441   YZ=             -0.0364
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             46.9682  YYY=              0.5799  ZZZ=             -2.8563  XYY=              8.6329
  XXY=             -0.3430  XXZ=             -3.1630  XZZ=              8.8307  YZZ=             -0.9595
  YYZ=             -0.8253  XYZ=             -0.0651
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1527.9048 YYYY=            -75.9569 ZZZZ=            -90.7041 XXXY=              8.9431
 XXXZ=             84.4509 YYYX=              6.6847 YYYZ=             -0.7075 ZZZX=             70.0484
 ZZZY=             -1.0038 XXYY=           -244.4213 XXZZ=           -248.3426 YYZZ=            -27.7749
 XXYZ=             -0.2827 YYXZ=             23.2037 ZZXY=              3.1430
 N-N= 1.705633794158D+02 E-N=-7.694575792391D+03  KE= 3.070061162612D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE386\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.346074,-0.000064,0.000117\H,0,-1.026028,-0.876411,-0.545947\H,0
 ,-1.02596,0.911004,-0.485765\H,0,-1.026089,-0.034819,1.032105\Br,0,1.6
 66279,0.000011,-0.000014\Cl,0,-3.209855,0.000004,-0.000021\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.4008376\RMSD=6.859e-09\Dipole=-1.604
 8741,-0.0000531,0.0000871\Quadrupole=-15.3088094,7.6543945,7.6544149,0
 .0000057,-0.0000661,-0.0000108\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 YOU SHOULDN'T LET PEOPLE DRIVE YOU CRAZY
 WHEN YOU KNOW IT'S WITHIN WALKING DISTANCE.
 Job cpu time:       0 days  0 hours  0 minutes 27.8 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.2 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:52:48 2020.
